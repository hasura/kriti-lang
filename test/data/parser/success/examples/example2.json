{
  "event": {
    "name": "Freddie Jones",
    "age": 27,
    "author": {
      "articles": [
        { "id": 0, "title": "The Elements", "length": 150, "published": true},
        { "id": 1, "title": "ARRL Handbook", "length": 1000, "published": true},
        { "id": 2, "title": "The Mars Trilogy", "length": 500, "published": false}
      ]
    }
  }
}
