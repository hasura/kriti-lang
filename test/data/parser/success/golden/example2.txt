Object
    ( fromList
        [
            ( "event"
            , Object
                ( fromList
                    [
                        ( "author"
                        , Object
                            ( fromList
                                [
                                    ( "articles"
                                    , Array
                                        [ Object
                                            ( fromList
                                                [
                                                    ( "length"
                                                    , Number 150.0
                                                    )
                                                ,
                                                    ( "title"
                                                    , String "The Elements"
                                                    )
                                                ,
                                                    ( "id"
                                                    , Number 0.0
                                                    )
                                                ,
                                                    ( "published"
                                                    , Boolean True
                                                    )
                                                ]
                                            )
                                        , Object
                                            ( fromList
                                                [
                                                    ( "length"
                                                    , Number 1000.0
                                                    )
                                                ,
                                                    ( "title"
                                                    , String "ARRL Handbook"
                                                    )
                                                ,
                                                    ( "id"
                                                    , Number 1.0
                                                    )
                                                ,
                                                    ( "published"
                                                    , Boolean True
                                                    )
                                                ]
                                            )
                                        , Object
                                            ( fromList
                                                [
                                                    ( "length"
                                                    , Number 500.0
                                                    )
                                                ,
                                                    ( "title"
                                                    , String "The Mars Trilogy"
                                                    )
                                                ,
                                                    ( "id"
                                                    , Number 2.0
                                                    )
                                                ,
                                                    ( "published"
                                                    , Boolean False
                                                    )
                                                ]
                                            )
                                        ]
                                    )
                                ]
                            )
                        )
                    ,
                        ( "name"
                        , String "Freddie Jones"
                        )
                    ,
                        ( "age"
                        , Number 27.0
                        )
                    ]
                )
            )
        ]
    )